poll/public/js/translations/fr/textjs.js

```diff
@@ -4,36 +4,90 @@
                   init: function() {
                     
 
-(function (globals) {
+(function(globals) {
 
   var django = globals.django || (globals.django = {});
 
   
-  django.pluralidx = function (count) { return (count == 1) ? 0 : 1; };
+  django.pluralidx = function(count) { return (count == 1) ? 0 : 1; };
   
 
-  
-  /* gettext identity library */
+  /* gettext library */
 
-  django.gettext = function (msgid) { return msgid; };
-  django.ngettext = function (singular, plural, count) { return (count == 1) ? singular : plural; };
-  django.gettext_noop = function (msgid) { return msgid; };
-  django.pgettext = function (context, msgid) { return msgid; };
-  django.npgettext = function (context, singular, plural, count) { return (count == 1) ? singular : plural; };
+  django.catalog = django.catalog || {};
   
-
-  django.interpolate = function (fmt, obj, named) {
-    if (named) {
-      return fmt.replace(/%\(\w+\)s/g, function(match){return String(obj[match.slice(2,-2)])});
-    } else {
-      return fmt.replace(/%s/g, function(match){return String(obj.shift())});
-    }
+  var newcatalog = {
```

Comment:
It seems to be missing some translations, is this expected?
![image](https://user-images.githubusercontent.com/5691347/48093493-a61e2d00-e1f6-11e8-82f5-5e0e0ec2fb84.png)